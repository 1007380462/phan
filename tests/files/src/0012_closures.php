<?php

function f() : int {
    $alpha = 42;

    $closure = function(int $beta) use ($alpha) {
        return $alpha + $beta;
    };

    return $closure(42);
}
