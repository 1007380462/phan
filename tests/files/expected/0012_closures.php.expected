%s:13 VarError Variable $c is not defined
